sdk/communication/AzureCommunicationChat/Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
     dependencies: [
         .package(
             url: "https://github.com/Azure/azure-sdk-for-ios.git",
-            .branch("master")
+            .revision("3e3c80d60173613c8dd4cb6b219188cf5070e8e7")
```

Comment:
Is this change intended to be merged?

Reply:
Yes, though we may need to change it to something else like the release tag? I'm curious how either can really even matter since if you download AzureSDK, AzureCore is baked in.